### DISCUSSION TITLE
TaxCategoryCodes innerhalb der EU

### DISCUSSION THREAD

**Discussion by BBessler** [stephanstapel/ZUGFeRD-csharp#508] [Q&A]
Hallo, 

ich hab jetzt valide XMLs für Deutschland erzeugt.
Leider habe ich noch das Problem das ich kein validen Zugferd-XMLs erzeuge, wenn die Lieferung in die EU nicht Deutschlands habe.
Mir ist nicht wirklich klar was ich setzten muss. 
Solbald ich in den InvoiceLiens und ApplicableTradeTax TaxCategoryCodes.S mit wert 0 setze, ist dies ungültig. Dann habe ich TaxCategoryCodes.E gesetzt das passt aber auch nicht. Gar nicht setzten ist auch nicht ok.
Wie wäre das richtige vorgehen?

Herzlichen Dank

**Comment by m4Viper** [comment]
Servus,
zeigt dir dein Validator nicht an welche Felder nicht stimmen und was der richtige Wert in etwa wäre?
Hat mir meist sehr weitergeholfen.

Für EU (intra-community) brauchst du:
taxcode = TaxCategoryCodes.K;
taxtype = TaxTypes.VAT;
taxexemption = TaxExemptionReasonCodes.VATEX_EU_IC;
